src/ftest.jl

```diff
@@ -121,21 +102,47 @@ function show{N}(io::IO, ftr::FTestResult{N})
 
     nc = 10
     nr = N
-    outrows = Array{String, 2}(nr+1, nc)
+    outrows = Matrix{String}(nr+1, nc)
     
-    outrows[1, :] = ["", "Res. DOF",  "DOF",  "ΔDOF",  "SSR",
-                    "ΔSSR",  "R²",  "ΔR²",  "F*",  "p(>F)"]
-    outrows[2, :] = ["Model 1", @sprintf("%.4f", ftr.dof_resid[1]),
-                     @sprintf("%.4f", ftr.nparams[1]), " ", @sprintf("%.4f", ftr.ssr[1]),
-                     " ", @sprintf("%.4f", ftr.r2[1]), " ", " ", " "]
+    outrows[1, :] = [
+                     "",
+                     "Res. DOF",
+                     "DOF",
+                     "ΔDOF",
+                     "SSR",
+                     "ΔSSR",
+                     "R²",
+                     "ΔR²",
+                     "F*",
+                     "p(>F)"
+                     ]
+
+    outrows[2, :] = [
```

Comment:
Here too you can put an item on the first line, and the closing bracket with the last element. Same below.